OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -49,6 +57,10 @@ public class AircraftInfo : ITraitInfo, IPositionableInfo, IFacingInfo, IMoveInf
 			"TakeOffOnCreation = Take off from creator when spawned.")]
 		public readonly FlightDynamic FlightDynamics = FlightDynamic.TakeOffOnCreation | FlightDynamic.MoveIntoShroud;
 
+		[Desc("Behavior when aircraft becomes idle. Options are Land, ReturnToBase, LeaveMap, and None.",
+			"'None' will make the aircraft hover (if it has Hover FlightDynamic) or fly in circles and also serves as fallback.")]
```

Comment:
IMO it would be clearer if we changed this second line to something like "'Land' will behave like 'None' (hover or circle) if a suitable landing site is not available.".